core/tests/puppeteer-acceptance-tests/user-utilities/voiceover-admin-utils.ts

```diff
@@ -0,0 +1,150 @@
+
+const updatedVoiceoverArtist = 'div.e2e-test-voice-artist-voiceoverartist';
+
+export class VoiceoverAdmin extends BaseUser {
+  /**
+   * Function to navigate to creator dashboard page
+   */
+  async navigateToCreatorDashboardPage(): Promise<void> {
+    await this.page.goto(creatorDashboardPage);
+  }
+
+  /**
+   * Function to navigate to exploration settings tab
+   */
+  async navigateToExplorationSettingsTab(): Promise<void> {
+    await this.page.waitForTimeout(500);
+    await this.clickOn(explorationSettingsTab);
+    await this.page.waitForTimeout(500);
+  }
+
+  /**
+   * Function to create exploration with title
+   * @param explorationTitle - title of the exploration
+   */
+  async createExplorationWithTitle(): Promise<void> {
+    await this.clickOn(createExplorationButton);
+    await this.page.waitForSelector(
+      `${dismissWelcomeModalSelector}:not([disabled])`
+    );
+    await this.clickOn(dismissWelcomeModalSelector);
+    await this.page.waitForTimeout(500);
+    await this.clickOn(textStateEditSelector);
+    await this.page.waitForTimeout(500);
+    await this.type(richTextAreaField, 'First Exploration');
+    await this.clickOn(saveContentButton);
+
+    await this.clickOn(addInteractionButton);
+    await this.clickOn(interactionEndExplorationInputButton);
+    await this.clickOn(saveInteractionButton);
+    await this.page.waitForSelector(`${saveChangesButton}:not([disabled])`);
+    await this.clickOn(saveChangesButton);
+    await this.clickOn(saveDraftButton);
+
+    await this.page.waitForSelector(
+      `${publishExplorationButton}:not([disabled])`
+    );
+    await this.clickOn(publishExplorationButton);
+    await this.type(explorationTitleInput, 'First Exploration');
+    await this.type(explorationGoalInput, 'First Exploration');
+    await this.clickOn(explorationCategoryDropdown);
+    await this.clickOn('Algebra');
+    await this.clickOn(saveExplorationChangesButton);
+    await this.page.waitForSelector(
+      `${publishExplorationButton}:not([disabled])`
+    );
+    await this.clickOn(publishExplorationButton);
+    await this.clickOn(explorationConfirmPublishButton);
+    await this.page.waitForSelector(closeShareModalButton);
+    await this.clickOn(closeShareModalButton);
+    await this.page.waitForTimeout(500);
+  }
+
+  /**
+   * Function to edit voiceover artist
+   */
+  async editVoiceoverArtist(artistUsername: string): Promise<void> {
+    await this.page.waitForSelector(editVoiceoverArtistButton);
+    await this.clickOn(editVoiceoverArtistButton);
+    await this.page.waitForSelector(voiceArtistEditSelector);
+    await this.type(voiceArtistEditSelector, artistUsername);
+    await this.clickOn(saveVoiceoverArtistEditButton);
+  }
+
+  /**
+   * Function to expect to see error toast message
+   */
+  async expectToSeeErrorToastMessage(): Promise<void> {
+    await this.page.waitForSelector(errorToastMessage);
+  }
```

Comment:
These `expect...` methods should make use of `showMessage` if the desired condition has been met, as well as `throw new Error('what went wrong...')` if the desired condition has not been or cannot be met.

Reply:
Updated the test.